website/pages/style.css

```diff
@@ -11,11 +9,10 @@
 @import '~@hashicorp/react-toggle/style.css';
 @import '~@hashicorp/react-alert-banner/style.css';
 @import '~@hashicorp/react-button/styles/index.css';
-@import '~@hashicorp/react-content/style.css';
-@import '~@hashicorp/react-docs-page/style.css';
 @import '~@hashicorp/react-featured-slider/style.css';
 @import '~@hashicorp/react-hero/style.css';
 @import '~@hashicorp/react-logo-grid/style.css';
+@import '~@hashicorp/react-search/style.css';
```

Comment:
Previously `react-docs-page` was responsible for this import, so it wasn't necessary here. Now that `docs-page` has been converted to CSS modules, this import is required.